src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -1,23 +1,23 @@
-package seedu.address.logic.commands;
-
-import static java.util.Objects.requireNonNull;
-
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-
-/**
- * Clears the address book.
- */
-public class ClearCommand extends Command {
-
-    public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
-
-
-    @Override
-    public CommandResult execute(Model model) {
-        requireNonNull(model);
-        model.setAddressBook(new AddressBook());
-        return new CommandResult(MESSAGE_SUCCESS);
-    }
-}
+//package seedu.address.logic.commands;
+//
+//import static java.util.Objects.requireNonNull;
+//
+//import seedu.address.model.AddressBook;
+//import seedu.address.model.Model;
+//
+///**
+// * Clears the address book.
+// */
+//public class ClearCommand extends Command {
+//
+//    public static final String COMMAND_WORD = "clear";
+//    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+//
+//
+//    @Override
+//    public CommandResult execute(Model model) {
+//        requireNonNull(model);
+//        model.setAddressBook(new AddressBook());
+//        return new CommandResult(MESSAGE_SUCCESS);
+//    }
+//}
```

Comment:
Is there a need to comment out unused classes?

Reply:
Unless you want to me to refactor all the dependencies? I still think keeping a old copy for ref is better.

Reply:
I see, no worries just wanted to know the reason. Good job!